ava.config.cjs

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   extensions: ["ts"],
   require: ["ts-node/register"],
+  timeout: '120s'
```

Comment:
I had issues with Docker tests timing out and for some reason ava just marks the tests as OK if it times out. :shrug: